.circleci/config.yml

```diff
@@ -22,15 +22,18 @@ commands:
     downstream:
         steps:
             - checkout
-            # - restore_cache:
```

Comment:
We had commented these out earlier to cache-bust. Although it doesn’t provide a significant benefit (just sub-minute run time improvements), I’ve re-added it.